spec/feature/confirm_email_prompt_spec.rb

```diff
@@ -0,0 +1,111 @@
+require "pry"
+RSpec.feature "Confirm email prompt" do
+  scenario "Seeing the banner on new user signup" do
```

Comment:
Could this be reworded to make it slightly clearer what it is tested, e.g. `Banner is present for user with unconfirmed email address`?